tests/Feature/BookTest.php

```diff
@@ -44,6 +45,53 @@ public function test_get_all_books(): void
                 });
         });
 }
+    public function test_get_all_claimed_books(): void
+    {
+        Claim::factory()->create();
+
```

Comment:
To add some more value to this test, you should also create an unclaimed book - to make sure it only gives you the claimed one

Reply:
Test updated!